airflow/providers/snowflake/provider.yaml

```diff
@@ -29,7 +29,7 @@ integrations:
     external-doc-url: https://snowflake.com/
     how-to-guide:
       - /docs/howto/operator/snowflake.rst
-    tags: [software]
+    tags: [service]
```

Comment:
I wanted to avoid creating a new page with only one operator. Earlier I was hoping that Yandex would add other integrations as well and then a separate section would make sense, but they didn't.